pandasdmx/utils/__init__.py

```diff
@@ -89,7 +89,13 @@ def __call__(self, name, fields):
         return namedtuple class as singleton 
         """
 
-        fields = tuple(fields)
+        _fields = []
+        for field in fields:
+            if "-" in field:
+                field = field.replace("-", "_")
+            _fields.append(field)

```

Comment:
Why is this required? It would make it slower. And shouldn't the model representation be neutral, i.e. as close to the source as possible?